src/libs/Notification/PushNotification/subscribeToReportCommentPushNotifications.ts

```diff
@@ -27,10 +29,27 @@ Onyx.connect({
  * Setup reportComment push notification callbacks.
  */
 export default function subscribeToReportCommentPushNotifications() {
-    PushNotification.onReceived(PushNotification.TYPE.REPORT_COMMENT, ({reportID, reportActionID, onyxData}) => {
+    PushNotification.onReceived(PushNotification.TYPE.REPORT_COMMENT, ({reportID, reportActionID, onyxData, lastUpdateID, previousUpdateID}) => {
         Log.info(`[PushNotification] received report comment notification in the ${Visibility.isVisible() ? 'foreground' : 'background'}`, false, {reportID, reportActionID});
-        Onyx.update(onyxData ?? []);
-        backgroundRefresh();
+
+        if (onyxData && lastUpdateID && previousUpdateID) {
+            Log.info('[PushNotification] reliable onyx update received', false, {lastUpdateID, previousUpdateID, onyxDataCount: onyxData?.length ?? 0});
+
+            const updates: OnyxUpdatesFromServer = {
+                type: CONST.ONYX_UPDATE_TYPES.PUSHER,
```

Comment:
I am not sure with we want to add a new `AIRSHIP` type here. Ultimately, we just want to apply the updates in the same way that we do with Pusher.

Reply:
Added anyway for the sake of logging. It should behave identically to pusher.